src/js/apps/admin/program/flow/flow_app.js

```diff
@@ -0,0 +1,49 @@
+import Radio from 'backbone.radio';
+
+import intl from 'js/i18n';
+
+import App from 'js/base/subrouterapp';
+
+import { LayoutView, ContextTrailView, HeaderView } from 'js/views/admin/program/flow/flow_views';
+import { SidebarView } from 'js/views/admin/program/sidebar/sidebar-views';
+
+export default App.extend({
+  onBeforeStart() {
+    this.showView(new LayoutView());
+    // this.getRegion('content').startPreloader();
+  },
+  beforeStart({ flowId, programId }) {
+    return [
+      Radio.request('entities', 'fetch:programFlows:model', flowId),
+      Radio.request('entities', 'fetch:programs:model', programId),
+    ];
+  },
+  onFail() {
+    Radio.request('alert', 'show:error', intl.admin.program.flowApp.notFound);
+    this.stop();
+  },
+  onStart({ currentRoute }, [flow], [program]) {
+    this.flow = flow;
```

Comment:
do we need to attach flow here? Looks like it's only being used on line 29 (unless I'm missing something...)

Reply:
I think we may end up using it elsewhere in the app once we get the action list implemented.